addons/mail/static/src/views/fields/statusbar_duration/statusbar_duration_field.js

```diff
@@ -13,7 +13,8 @@ export class StatusBarDurationField extends StatusBarField {
         const durationTracking = this.props.record.data.duration_tracking || {};
         if (Object.keys(durationTracking).length) {
             for (const item of items) {
-                const duration = durationTracking[item.value];
+                let duration = durationTracking[item.value];
+                duration = (duration==0) ? 1 : duration;
```

Comment:
Actually I see in formatDuration this :

```js
    if (seconds < 60) {
        seconds = 60;
    }
```

So we could just keep old code here but just use `if (duration >= 0)` instead of `if (duration > 0)` :)

Reply:
Updated the code as per your suggestions. 
Thanks!!